apps/frontend/src/app/5_pages/AaveReserveOverviewPage/components/BorrowDetailsGraph/components/Chart/Chart.tsx

```diff
@@ -8,15 +8,14 @@ import {
   GRID_COLOR,
   TICK_COLOR,
 } from './Chart.constants';
-import { MockData } from './Chart.types';
+import { InputData } from './Chart.types';
 import { htmlLegendPlugin } from './Chart.utils';
 
 type ChartProps = {
-  mockData: MockData<any>;
-  yLabel1: string;
+  input: InputData<any>;
```

Comment:
Could we know the type to replace this any?

Reply:
In other similar type above, you use it a generic type. I like it more than the any. What do u think?